snuba/web/db_query.py

```diff
@@ -186,7 +195,11 @@ def execute_query(
 
     timer.mark("execute")
     stats.update(
-        {"result_rows": len(result["data"]), "result_cols": len(result["meta"])}
+        {
+            "result_rows": len(result["data"]),
+            "result_cols": len(result["meta"]),
+            "max_threads": clickhouse_query_settings.get("max_threads", None),
```

Comment:
keeping track of what the max threads we ran the query with, needed it for testing, could be useful for other things too